backend/dal/Helpers/Extensions/ProjectStatusExtensions.cs

```diff
@@ -0,0 +1,48 @@
+using Pims.Dal.Entities;
+
+namespace Pims.Dal.Helpers.Extensions
+{
+    public static class ProjectStatusExtensions
+    {
+        public static bool StatusMatch(this ProjectStatus projectStatus, ProjectStatusNames match)
+        {
+            return projectStatus.Id == (int) match;
+        }
+
+        public static bool IsDenied(this ProjectStatus projectStatus)
+        {
+            return projectStatus.StatusMatch(ProjectStatusNames.Denied);
+        }
+
+        public static bool IsDraft(this ProjectStatus projectStatus)
+        {
+            return projectStatus.StatusMatch(ProjectStatusNames.Draft);
+        }
+
+        public static bool IsApproved(this ProjectStatus projectStatus)
+        {
+            return projectStatus.StatusMatch(ProjectStatusNames.Approved);
+        }
+
+        public static bool IsSubmitted(this ProjectStatus projectStatus)
+        {
+            return projectStatus.StatusMatch(ProjectStatusNames.Submitted);
+        }
+
+        public enum ProjectStatusNames
```

Comment:
Don't introduce an enum for managing project statuses.  I choose not to lock Project Statuses to an enum so that we could avoid the pitfalls associated with this approach.  I am planning in the next sprint to introduce `ProjectStatus.Code` to use for this kind of filtering.

Reply:
You can leave for now, but I'll have to remove next sprint.